AuthenticationDemo/AppDelegate.swift

```diff
@@ -75,7 +74,10 @@ extension AppDelegate {
         let loginConfiguration = LoginViewConfiguration(logoImage: UIImage(named: "default")!)
         let signupConfiguration = SignupViewConfiguration(termsAndServicesURL: URL(string: "https://www.wolox.com.ar/")!,
                                                           showLoginProviders: true)
-        let loginProviders: [LoginProvider] = [FacebookLoginProvider(), ExampleFailLoginProvider(), ExampleSuccessLoginProvider()]
+        let loginProviders: [LoginProvider] = [FacebookLoginProvider(),
+                                               GoogleLoginProvider(with: "223197056946-ssnoko9vqn23960ia31kt3cevg44d4m4.apps.googleusercontent.com"),
```

Comment:
Move it to config files or something. (So we show how this should be done with good practices.)